nucliadb/src/nucliadb/ingest/orm/brain_v2.py

```diff
@@ -255,8 +267,16 @@ def apply_field_paragraphs(
         unique_paragraphs: set[str] = set()
         user_paragraph_classifications = self._get_paragraph_user_classifications(user_field_metadata)
         paragraph_pages = ParagraphPages(page_positions) if page_positions else None
+
         # Splits of the field
-        for subfield, field_metadata in field_computed_metadata.split_metadata.items():
+
+        # Used to adjust the paragraph start/end when indexing splits, as they are all
+        # concatenated in the main text part of the brain Resource.
+        split_offset = 0
+        for subfield in self.sorted_splits(extracted_text):
+            if subfield not in field_computed_metadata.split_metadata:
+                continue
```

Comment:
Probably this shouldn't happen but if you find a split in the `extracted_text` and not in `split_metadata`, you still added it to the aggregated text and not taking into account its offset